examples/tutorials/advance_model_config/03_train_steps.ipynb

```diff
@@ -0,0 +1,446 @@
+{
```

Comment:
<p>what is an embedding? Do you really need to talk about it here? If yes, please tell few words about it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>